pkg/api/handlers/brokers_handler.go

```diff
@@ -46,3 +50,33 @@ func (bh *BrokerHandler) HandleGet() rest.Handler {
 	}
 	return rest.Handler(handler)
 }
+
+// KafkaCreateHandler is the function that processes requests at the /brokers/kafka endpoint
+func (bh *BrokerHandler) KafkaCreateHandler() rest.Handler {
```

Comment:
In this function, whenever there's an error I believe you should put a `return` after each `rest.Error(...)` call, so it's interrupted. (if what I said is correct, please do the same for the `HandleGet()` above)

Reply:
my bad, slipped my mind. will do